pages/api/tell-me/synchronize.js

```diff
@@ -22,51 +24,31 @@ async function createSynchronization(req, success, info) {
   })
 }
 
-async function createContributor(submission, req) {
+/**
+ * @param {import('../../../api/helpers/tell-me/contributorParsing').FormattedContributor} contributor
+ * @param {*} req
+ * @returns
+ */
+async function createContributor(contributor, req) {
```

Comment:
I'm not sur to understand the purpose of creating a separate function to run what's already a function.

Reply:
Indeed, useless ; probably an artefact of something I've done then undone ;)